schema/metrics/AggregatingMetric.go

```diff
@@ -0,0 +1,39 @@
+package metrics
```

Comment:
How liveness / readyness work:
Prometheus module has methods to register liveness / readyness monitor. Each registered monitor work independently. Prometheus module should aggregate state of all registered metric and show common result vie two approach: 0 / 1 into Prometheus th2_liveness / th2_readiness metric, create or delete /tmp/healthy /tmp/ready